docker/prod/indexer/indexer.service

```diff
@@ -15,48 +15,38 @@ SEARCH_SERVER_CHILL_TIME_BETWEEN_RUNS = 900
 
 RECORDING_INDEX_MAX_AGE = 60 * 60 * 24
 
+
+def lastest_index_dir(indexes_dir, version, index_type):
+    topdir = os.path.join(indexes_dir, version)
+    dirs = [f for f in os.listdir(topdir) if os.path.isdir(os.path.join(topdir, f))]
+    dirs.sort(reverse=True)
```

Comment:
Here is the current structure, as you can see under `data/1` directories are named with timestamps: 

```
/home/search/data/1
/home/search/data/1/1492679579
/home/search/data/1/1492679579/place_index
/home/search/data/1/1492679579/label_index
/home/search/data/1/1492679579/url_index
/home/search/data/1/1492679579/tag_index
/home/search/data/1/1492679579/cdstub_index
/home/search/data/1/1492679579/editor_index
/home/search/data/1/1492679579/artist_index
/home/search/data/1/1492679579/event_index
/home/search/data/1/1492679579/series_index
/home/search/data/1/1492679579/instrument_index
/home/search/data/1/1492679579/recording_index
/home/search/data/1/1492679579/work_index
/home/search/data/1/1492679579/releasegroup_index
/home/search/data/1/1492679579/area_index
/home/search/data/1/1492679579/annotation_index
/home/search/data/1/1492679579/release_index
/home/search/data/1/1492681520
/home/search/data/1/1492681520/place_index
/home/search/data/1/1492681520/label_index
/home/search/data/1/1492681520/url_index
/home/search/data/1/1492681520/tag_index
/home/search/data/1/1492681520/cdstub_index
/home/search/data/1/1492681520/editor_index
/home/search/data/1/1492681520/artist_index
/home/search/data/1/1492681520/event_index
/home/search/data/1/1492681520/series_index
/home/search/data/1/1492681520/instrument_index
/home/search/data/1/1492681520/recording_index
/home/search/data/1/1492681520/work_index
/home/search/data/1/1492681520/releasegroup_index
/home/search/data/1/1492681520/area_index
/home/search/data/1/1492681520/annotation_index
/home/search/data/1/1492681520/release_index
```